fe/fe-core/src/main/java/org/apache/doris/nereids/processor/pre/EliminateLogicalSelectHint.java

```diff
@@ -72,5 +72,19 @@ private void setVar(SelectHint selectHint, StatementContext context) {
                 }
             }
         }
+        // if sv set enable_nereids_planner=true and hint set enable_nereids_planner=false, we should set
+        // enable_fallback_to_original_planner=true and revert it after executing.
+        // throw exception to fall back to original planner
+        if (!sessionVariable.isEnableNereidsPlanner()) {
+            String key = "enable_fallback_to_original_planner";
```

Comment:
we should use `org.apache.doris.qe.SessionVariable#ENABLE_FALLBACK_TO_ORIGINAL_PLANNER`